lib/api/apiUtils/bucket/bucketEncryption.js

```diff
@@ -108,7 +109,31 @@ function parseBucketEncryptionHeaders(headers) {
     };
 }
 
+function checkExpectedBucketOwner(headers, bucket, log, cb) {
```

Comment:
Why do we check this header specifically for delete-bucket-encryption? According to https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-owner-condition.html, it looks to me this header is generic to pretty much all object and bucket operations, and optional, so kinda looks orthogonal to bucket encryption.

Reply:
It was accidentally skipped in put/get. I've created a ticket to add support for them.

Reply:
OK sounds good

Ideally I think it should not be defined in a bucket-encryption-specific file but it should be part of the common checks path for all request types.

Reply:
Added a note in the ticket to look at moving it.